src/components/Advertisements/Advertisements.spec.tsx

```diff
@@ -1,4 +1,7 @@
 import React, { act } from 'react';
+import { describe, test, expect, vi } from 'vitest';
+import '@testing-library/jest-dom';
```

Comment:
Why was this included if we are migrating away from jest?

Reply:
I apologize for my confusion. i have done changes